src/app/search/search-documents-table-rows/search-document-table-rows.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, Input, Output, OnInit, OnDestroy, EventEmitter, ChangeDetectorRef } from '@angular/core';
+import { Subject } from 'rxjs';
+import { TableComponent } from 'app/shared/components/table-template/table.component';
+import { TableObject } from 'app/shared/components/table-template/table-object';
+import { Router, ActivatedRoute } from '@angular/router';
+import { ApiService } from 'app/services/api';
+import { Utils } from 'app/shared/utils/utils';
+
+@Component({
+  selector: 'tbody[app-document-table-rows]',
+  templateUrl: './search-document-table-rows.component.html',
+  styleUrls: ['./search-document-table-rows.component.scss']
+})
+
+export class DocSearchTableRowsComponent implements OnInit, OnDestroy, TableComponent {
+  private ngUnsubscribe: Subject<boolean> = new Subject<boolean>();
```

Comment:
For readability, Angular classes usually declare properties under the Input and Output decorators.

Reply:
makes sense, cleaner that way